packages/@dcos/sorting/__tests__/compare-as-string-test.ts

```diff
@@ -0,0 +1,30 @@
+import { compareAsString, ComparisonResult } from "../";
+
+const idLense = (x: any) => x;
+
+describe("compareAsString", () => {
+  it("returns 0 if any on the values is null", () => {
+    expect(compareAsString(idLense)(null, "x")).toBe(ComparisonResult.EQUAL);
+    expect(compareAsString(idLense)("n", null)).toBe(ComparisonResult.EQUAL);
+  });
+
+  it("compares strings", () => {
+    expect(compareAsString(idLense)("a", "b")).toBe(ComparisonResult.SMALLER);
+    expect(compareAsString(idLense)("b", "a")).toBe(ComparisonResult.BIGGER);
+    expect(compareAsString(idLense)("b", "b")).toBe(ComparisonResult.EQUAL);
+  });
+
+  it("converts unknown type to strings before comparing", () => {
```

Comment:
I think a more meaningful test could be:
`"9" > "100" is bigger` because than it contradicts common knowledge

Reply:
Had a hard time coming up with tests, but I like this one ;)